tests/config/test_argparse_config.py

```diff
@@ -64,7 +64,7 @@ def test_old_names() -> None:
         run = Run([EMPTY_MODULE, "--ignore=test,test_two"], exit=False)
         assert run.linter.config.ignore == ["test", "test_two"]
         assert run.linter.config.ignore == run.linter.config.black_list
-        assert run.linter.config.ignore_patterns == [re.compile("^\\.#")]
+        assert list(run.linter.config.ignore_patterns) == [re.compile("^\\.#")]
```

Comment:
```suggestion
        assert run.linter.config.ignore_patterns == (re.compile("^\\.#"),)
```

Should be this then?